Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -100,6 +102,9 @@ void function GamemodeFD_Init()
 
 	//earn meter
 	ScoreEvent_SetupEarnMeterValuesForMixedModes()
+
+	//Data Collection
+	AddStunLaserHealCallback( FD_HealTeammate )
```

Comment:
We will probably want another callback for amped batteries (maybe) and monarch esmoke heals, but this is a good start

Reply:
Finished